schema.sql

```diff
@@ -10,3 +10,30 @@ CREATE TABLE animals (
 );
 
 ALTER TABLE animals ADD species VARCHAR(255);
+
+/*Create owners table*/
+
+CREATE TABLE owners (
+    id serial PRIMARY KEY,
+    full_name VARCHAR(255),
+    age SMALLINT
```

Comment:
- [ ] Kindly use the data type `INT` as shown in the project requirements. It is important for us as developers to closely follow given requirements.